pycaret/internal/ts_tests.py

```diff
@@ -0,0 +1,227 @@
+from typing import Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+
+def find_trend(
+    data: np.ndarray,
+    trend_order: int = 1,
+    **kwargs
+):
+    coeffs = np.polyfit(range(len(data)), data, trend_order)
+    slope = coeffs[-2]
+    return slope
+
+
+def trend_and_seasonality_test(
+    ts: Union[pd.DataFrame, pd.Series],
+    seasonal_periods=(3, 5, 7, 12, 14, 30, 31),
```

Comment:
Let's remove the default values from here. This should come from outside depending on the time series.